### PR TITLE
Added my information to the contributors section to the about.html

### PR DESCRIPTION
**Pull request by stu7**

### CODE REVIEW

File: docs/about.html [hunk 1]
```diff
@@ -78,6 +78,7 @@ <h2 id="organizers-volunteers">Contibutors</h2>
                     <ul>
                         <li><a href="https://github.com/bryanrigsby" target="_blank" title="Bryan Rigsby" rel="external">Bryan Rigsby</a></li>
                         <li><a href="https://github.com/mstub" target="_blank" title="Matt Stubenhofer" rel="external">Matt Stubenhofer</a></li>
+                        <li><a href="https://github.com/stu7" target="blank" title="Norman Haase" rel="external">Norman Haase</a></li>
```
Comment (CarsonMcKinstry) [inline docs/about.html]: target attribute in your `<a>` won't open a new tab/window.

**Review by CarsonMcKinstry — DISMISSED** [review]
Good starting point!
You might take a look at some of the issues in this repo, see if it is something you can do, and then let me know when this PR is ready.

**Comment by stu7** [comment]
Are you looking for fixes from issues added to this initial PR?

**Comment by stu7** [comment]
Initial target attribute change was made in this initial PR.  PR is ready.

**Comment by stu7** [comment]
I'll continue to work through on practice issues here, correct any errors and push them back out

File: docs/about.html (hunk 1 above)
Comment (stu7) [reply docs/about.html]: Fixed this error

**Comment by stu7** [comment]
Additions changes were made to the index.html and projects.html in this initial PR. PR is ready for review.

**Review by dashcraft — CHANGES_REQUESTED** [review]
There is an opened ul tag by the Techlahoma reference. Can you please close it or remove it, if it is unnecessary.

**Comment by mattstub** [comment]
working on cleaning this up in a new PR

**Pull request closed without merging**